components/contribution-flow/index.js

```diff
@@ -201,7 +201,7 @@ class ContributionFlow extends React.Component {
                 : null,
             customData: stepDetails.customData,
             paymentMethod: await this.getPaymentMethod(),
-            platformContributionAmount: getGQLV2AmountInput(stepDetails.platformContribution, undefined),
+            platformTipAmount: getGQLV2AmountInput(stepDetails.platformContribution, undefined),
```

Comment:
As it's really sensible, can you point me again on where it's supported in the API?

Reply:
This is where we added the `platformTipAmount` when creating the order; 

https://github.com/opencollective/opencollective-api/blob/58e91edad9f836b7fb1fc913ea79517a2fa2e450/server/graphql/v2/mutation/OrderMutations.js#L72

Also just to note this is merged we have one final PR; https://github.com/opencollective/opencollective-api/pull/6980 to rebase and merge to remove the older platformContributionAmount